javascript/javascript3/week2/badMovies.js

```diff
@@ -0,0 +1,25 @@
+const movie_Api = 'https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json'
+const badTitlesAfter2000 = []
+let ali = []
```

Comment:
It is not clear what is this for. consider removing if it is not used or rename it.